rust/Cargo.toml

```diff
@@ -45,7 +45,7 @@ version = "0.8.1"
 
 [dependencies.mediasoup-sys]
 path = "../worker"
-version = "0.5.4"
+version = "0.5.5"
```

Comment:
This will not work, you also need to bump the version of `mediasoup-sys`, this simply defines minimum requirement. Also let's bump it to `0.6.0` too.